Makefile

```diff
@@ -210,37 +212,65 @@ docker-image-no-clean: GIT_VERSION
 		--build-arg LOCKDEBUG=${LOCKDEBUG} \
 		--build-arg V=${V} \
 		--build-arg LIBNETWORK_PLUGIN=${LIBNETWORK_PLUGIN} \
-		-t "cilium/cilium:$(DOCKER_IMAGE_TAG)" .
+		-t "cilium/cilium:$(DOCKER_IMAGE_TAG)-${ARCH}" .
 	$(QUIET)echo "Push like this when ready:"
-	$(QUIET)echo "${CONTAINER_ENGINE} push cilium/cilium:$(DOCKER_IMAGE_TAG)"
+	$(QUIET)echo "${CONTAINER_ENGINE} push cilium/cilium:$(DOCKER_IMAGE_TAG)-${ARCH}"
+
+docker-cilium-manifest:
+	@$(ECHO_CHECK) contrib/scripts/push_manifest.sh cilium $(DOCKER_IMAGE_TAG)
```

Comment:
This makefile is getting a little bit out of hand :-) @joestringer maybe we should have a Makefile dedicated to build docker images? what do you think?

Reply:
This sounds like a reasonable suggestion to me. I don't mind merging first then refactoring the docker make targets out as a subsequent step though.

Reply:
This sounds like a reasonable suggestion to me. I don't mind merging first then refactoring the docker make targets out as a subsequent step though. In fact, doing this might make it easier to detect any regressions if we missed something.

Reply:
Would love to discuss how we can improve it!